### ISSUE TITLE
Cannot connect to the Server with Stomp message

### ISSUE THREAD

**Issue by adibend** [Fyro-Ing/JMeter-WebSocket-StompSampler#4]
Hi,

I'm trying to connect to our server with the websocket stomp message with no success.
I tried CONNECT\nlogin:gl0\nheart-beat:10000,10000\naccept-version:1.2\nhost: 1.9.12.67
as well with no success.

I get a green step but nothing moves to the server (traced by Wireshark)

What am i doing wrong?

Regards, Adi

Stomp Send message:

![stomp](https://user-images.githubusercontent.com/19553164/32772574-f05281be-c92e-11e7-8ce3-7e39f83df3d2.png)

Stomp answer:
![stompanswer](https://user-images.githubusercontent.com/19553164/32772706-5b92ff94-c92f-11e7-9a14-1198648bf3bf.png)

**Comment by Fyro-Ing** [comment]
You need to write inline message like ''CONNECT\naccept-version:1.1,1.0\nheart-beat:10000,10000", see #1 

or you need to modify source code like #3

**Comment by adibend** [comment]
Hi,

Thanks for the reply.
I tried #1 - it is not working and STOMP doesn't recognize the message.
If i want to try #3 - Where do i write the code in Jmeter?

Regards, Adi.

**Comment by Fyro-Ing** [comment]
see last comment from kazexd in #3

**Comment by mtytester** [comment]
Hi, 
Is there a way to use SEND method on the websocketstomp sampler?
Thanks,

**Comment by Fyro-Ing** [comment]
Have you try with "subscribe message panel" ?

**Comment by mtytester** [comment]
Yes I have tried it, but I had no success, do you think there is a workaround?
